postmarketOS/pmbootstrap
# Fix: Native arch APKINDEX downloaded twice

Nowadays pmb.config.build_device_architectures holds the native
architecture as well, so we don't need to explicitly download the
native architecture APKINDEX.

(We're still downloading for more arches than necessary, most people don't need the `x86` index for example. That could be improved with a follow-up PR.)

### How to test
* Run `pmbootstrap update` once without and once with this PR.
* It should download fewer APKINDEX files
* In `pmbootstrap log` you should see that it doesn't download the native arch twice with the PR
